fesvr/htif.cc

```diff
@@ -142,7 +142,8 @@ std::map<std::string, uint64_t> htif_t::load_payload(const std::string& payload,
   } preload_aware_memif(this);
 
   try {
-    return load_elf(path.c_str(), &preload_aware_memif, entry, expected_xlen);
+    return load_elf(path.c_str(), &preload_aware_memif, entry,
```

Comment:
The implication is that `preload_aware_memif` does not know the target endianness--is that really true, and should it be?

Reply:
Ah, I see it's NULL in a later commit.  Can you instead construct a dummy memif that takes no action on read/write?  That would keep the ELF loader cleaner.